core/modules/system/system.admin.inc

```diff
@@ -1566,41 +1566,99 @@ function system_site_information_settings_submit($form, &$form_state) {
 function system_cron_settings($form, &$form_state) {
   global $base_url;
   $form['#config'] = 'system.core';
+  $config = config('system.core');
+
+  $form['last_run'] = array(
+    '#type' => 'help',
+    '#markup' => t('Last run: %cron-last ago.', array(
+      '%cron-last' => format_interval(REQUEST_TIME - state_get('cron_last')),)),
+  );
+
   $form['description'] = array(
     '#type' => 'help',
-    '#markup' => t('Cron takes care of running periodic tasks like checking for updates and indexing content for search.'),
+    '#markup' => t('Cron takes care of running periodic tasks like checking for updates and indexing content for search. The preferred way to run such tasks is by <a href="https://en.wikipedia.org/wiki/Cron">setting up native cron jobs</a>. If that is not an available option with your hosting provider, then the following methods are alternatives.'),
```

Comment:
I don't think we should be linking to wikipedia. In fact, I don't think any of this new text adds value. "The following" also includes the link thats necessary for native cron jobs, so I worry that this might confuse people more than it helps.

Reply:
OK, removed.